services/portal/dataUpload/dataUploadSequences.js

```diff
@@ -0,0 +1,55 @@
+const dataUploadQuestions = require('./dataUploadQuestions.js');
+const dataUploadTasks = require('./dataUploadTasks.js');
+const dataUploadProps = require('./dataUploadProps.js');
+
+/**
+ * dataUpload sequences
+ */
+module.exports = {
+  // Sequences are for an service to combine multiple tasks and questions
+  async checkUnmappedFilesAreInSubmissionPage(fileObjects, isReady) {
+    // goto '/submission' page and check file number and sizes are correct
+    dataUploadTasks.goToSubmissionPage();
+    const expectedFileCount = fileObjects.length;
+    let expectedFileTotalSize = 0;
+    if (isReady) {
+      expectedFileTotalSize = fileObjects.reduce((acc, cur) => {
+        return acc + cur.fileSize;
+      }, 0);
+    }
+    await dataUploadQuestions.isNumberAndSizeOfUnmappedFilesCorrect(expectedFileCount, expectedFileTotalSize);
+  },
+
+  async checkUnmappedFilesAreNotInFileMappingPage(fileObjects) {
+    // goto '/submission/files' page and check all files are not displayed
+    dataUploadTasks.goToMapFilesPage();
+    const unexpectedFileNames = fileObjects.reduce((acc, cur) => {
+      acc.push(cur.fileName);
+      return acc;
+    }, []);
+    await dataUploadQuestions.cannotSeeUnmappedFilesOnPage(unexpectedFileNames);
+  },
+
+  async checkCouldMapFiles(fileObjects) {
```

Comment:
sorry this is a detail haha I think I would be easier to understand what this function does if the name was `mapFiles`